### PR TITLE
Build against Qt6.

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

**Comment by mkosola** [comment]
Ah, need to add ifdef to build properly

File: src/notification.cpp [hunk 1]
```diff
@@ -87,7 +87,7 @@ QDBusArgument &operator <<(QDBusArgument &argument, const NotificationImage &ima
     argument << image.hasAlphaChannel();
     argument << 8;
     argument << 4;
-    argument << QByteArray(reinterpret_cast<const char *>(image.bits()), image.byteCount());
+    argument << QByteArray(reinterpret_cast<const char *>(image.bits()), image.sizeInBytes());
```
Comment (Tomin1) [inline src/notification.cpp:91]: Doesn't exist in Qt 5.6.

https://doc.qt.io/qt-5/qimage.html#sizeInBytes

File: src/notification.cpp [hunk 2]
```diff
@@ -238,7 +238,7 @@ QVariantList decodeActionHints(const QList<NotificationData::ActionInfo> &action
             QVariantMap action;
 
             // Extract the element of the DBus call
-            QStringList elements(hint.split(' ', QString::SkipEmptyParts));
+            QStringList elements(hint.split(' ', Qt::SkipEmptyParts));
```
Comment (Tomin1) [inline src/notification.cpp:246]: Doesn't exist in Qt 5.6.

https://doc.qt.io/qt-5/qt.html#SplitBehaviorFlags-enum

**Review by Tomin1 — CHANGES_REQUESTED** [review]
This needs ifdefs.

File: src/notification.cpp [hunk 3]
```diff
@@ -87,7 +87,11 @@ QDBusArgument &operator <<(QDBusArgument &argument, const NotificationImage &ima
     argument << image.hasAlphaChannel();
     argument << 8;
     argument << 4;
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
```
Comment (Tomin1) [inline src/notification.cpp]: Should we have `5, 10, 0` instead of `6, 0, 0` though?

File: src/notification.cpp [hunk 4]
```diff
@@ -238,7 +242,11 @@ QVariantList decodeActionHints(const QList<NotificationData::ActionInfo> &action
             QVariantMap action;
 
             // Extract the element of the DBus call
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
```
Comment (Tomin1) [inline src/notification.cpp]: Should we have `5, 14, 0` instead of `6, 0, 0` though?

**Review by Tomin1 — COMMENTED** [review]
Looks better. I'm wondering about the version numbers though since those were introduced already before 6.0.

**Review by Tomin1 — APPROVED** [review]

**Review by pvuorela — APPROVED** [review]

**Pull request merged**